components/portal/widgets/partials/type__action-items.html

```diff
@@ -20,45 +20,46 @@
 -->
 <div class="widget-body action-items">
   <!-- No action items to display -->
-  <div layout="row" layout-align="center center" ng-show="actionItems.length == 0">
+  <div layout="row" layout-align="center center">
     <!-- Loading case -->
     <loading-gif data-object="actionItems" ng-show="loading"></loading-gif>
 
-    <!-- Error case (endpoint failure) -->
-    <div class="error" layout="row" layout-align="start center" ng-show="hasServiceError">
-      <div><md-icon class="md-warn">warning</md-icon></div>
-      <span>There was an issue loading the information. Click <a href="{{ widget.url }}" target="{{ widget.target ? widget.target : '_self' }}">{{ launchText }}</a> for more info.</span>
+    <div>
+      <md-list class="widget-list md-2-line">
+        <md-list-item aria-label="click to take action"
+          ng-repeat="item in actionItems | limitTo:itemsLimit"
+          ng-click="goToAction(item.actionUrl)" >
+          <div class="action-item__quantity">
+            <span>{{ item.quantity }}</span>
+          </div>
+          <div class="md-list-item-text">
+            <p ng-if="item.quantity == 1">{{ item.textSingular }}</p>
+            <p ng-if="item.quantity != 1">{{ item.textPlural }}</p>
+          </div>
+        </md-list-item>
+        <!-- Error case -->
+        <md-list-item  ng-if="actionItemErrors && actionItemErrors.length">
+          <md-icon class="md-warn action-item__error-quantity">warning</md-icon>
+          <span>
+            Something went wrong.
```

Comment:
I feel we don't we need the phrase "Something went wrong." prepended to every error. We're working with limited space, I think we can be straight to the point.